pages/fundamentals/introduction/index.pt-BR.md

```diff
@@ -0,0 +1,242 @@
+
+A web é um recurso cada vez mais importante em muitos aspectos da vida: educação, emprego, governo, comércio, assistência médica, recreação e muito mais. É essencial que a web seja acessível para fornecer acesso e oportunidades iguais a pessoas com habilidades diferentes. O acesso às tecnologias de informação e comunicação, incluindo a web, é definido como um direito humano básico na Convenção das Nações Unidas sobre os Direitos das Pessoas com Deficiência (UN [CRPD](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html)).
+
+A web oferece a possibilidade de acesso sem precedentes à informação e interação para muitas pessoas com deficiência. Ou seja, as barreiras de acessibilidade a materiais impressos, áudio e mídia visual podem ser superadas muito mais facilmente por meio de tecnologias web.
+
+A acessibilidade oferece suporte à inclusão social para pessoas com deficiência, bem como outras pessoas, como:
+
+-   pessoas mais velhas
+-   pessoas em áreas rurais
+-   pessoas em países em desenvolvimento
+
+**Há também um forte caso de negócio para acessibilidade.** Conforme mostrado na seção anterior, o design acessível melhora a experiência geral e a satisfação da pessoa usuária, especialmente em uma variedade de situações, em diferentes dispositivos e para pessoas usuárias mais velhas. A acessibilidade pode melhorar sua marca, impulsionar a inovação e estender seu alcance no mercado.
+
+A acessibilidade web é **exigida por lei** em muitas situações.
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Mais informações sobre como a acessibilidade é importante" class="simple aside" %}
+{:/}
+
+-   Informações gerais sobre benefícios para os negócios estão em [[O caso de negócio para acessibilidade digital]](/business-case/).
+    -   Exemplos dos benefícios de [tornar mídias de áudio e vídeo acessíveis](/media/av/) estão na seção [Benefícios para organizações](/media/av/users-orgs/#benefits).
+-   Orientações sobre como descobrir requisitos legais estão no recurso arquivado [Fatores legais e de política](https://www.w3.org/WAI/business-case/archive/pol).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## Tornando a web acessível {#making}
+
+A acessibilidade web depende de vários componentes trabalhando juntos, incluindo tecnologias web, navegadores web e outros \"agentes da pessoa usuária\", ferramentas de autoria e sites.
+
+A Iniciativa de Acessibilidade Web ([WAI](/about/participating/)) do W3C desenvolve especificações técnicas, diretrizes, técnicas e recursos de suporte que descrevem soluções de acessibilidade. Estes são considerados padrões internacionais para acessibilidade web; por exemplo, as <abbr title="Diretrizes de Acessibilidade de Conteúdo Web (WCAG)">WCAG</abbr> 2.0 também são um padrão <abbr title="Organização Internacional para Padronização">ISO</abbr>: ISO/<abbr title="Comissão Eletrotécnica Internacional">IEC</abbr> 40500.
```

Comment:
```suggestion
A Iniciativa de Acessibilidade Web ([WAI](/about/participating/)) do W3C desenvolve especificações técnicas, diretrizes, técnicas e recursos de suporte que descrevem soluções de acessibilidade. Estes são considerados padrões internacionais para acessibilidade web; por exemplo, as <abbr title="Diretrizes de Acessibilidade para Conteúdo Web">WCAG</abbr> 2.0 também são um padrão <abbr title="Organização Internacional para Padronização">ISO</abbr>: ISO/<abbr title="Comissão Eletrotécnica Internacional">IEC</abbr> 40500.
```

"Diretrizes de Acessibilidade para Conteúdo Web" is used in the [Authorized Translation of WCAG 2.0](https://www.w3.org/Translations/WCAG20-pt-br/) and the [Candidate Authorized Translation of WCAG 2.2](https://www.w3c.br/traducoes/wcag/wcag22-pt-BR/)